packages/lit-dev-server/src/middleware/content-security-policy-middleware.ts

```diff
@@ -233,6 +233,10 @@ export const contentSecurityPolicyMiddleware = (
       policy = strictFallbackCsp;
     }
     ctx.set('Content-Security-Policy', policy);
-    return next();
+    await next();
```

Comment:
Could move this above the `if`, and then it can be `if (ctx.path.endsWith('/') || ctx.status === 404) { ...`

Reply:
Beautiful!